pkg/interruptionevent/spot-itn-monitor_test.go

```diff
@@ -60,7 +60,9 @@ func TestMonitorForSpotITNEventsSuccess(t *testing.T) {
 			"Expected description to contain: "+startTime+" but is actually: "+result.Description)
 	}()
 
-	err := interruptionevent.MonitorForSpotITNEvents(drainChan, cancelChan, imds)
+	nodeName := "test-node"
+	spotITNMonitor := interruptionevent.NewSpotInterruptionMonitor(imds, drainChan, cancelChan, nodeName)
+	err := spotITNMonitor.Monitor()
```

Comment:
this is pedantic but should we change these function names? They're referencing functions that are removed. `TestMonitor` would be better imo

Reply:
the problem is that they're still in the same package, so those names would conflict

Reply:
I was thinking about moving these handler into individual sub packages... idk if that's too much though, but it would isolate them.  Ideally, they shouldn't share anything. wdut?

Reply:
That sounds good to me

Reply:
updated